modules/manage/pages/manage-xdcr/replicate-using-scopes-and-collections.adoc

```diff
@@ -121,6 +140,45 @@ The fields now appear as follows;
 +
 image::manage-xdcr/xdcr-replicate-to-103.png[,680,align=left]
 
+. Save the replication, by left-clicking on the *Save Replication* button.
++
+image::manage-xdcr/saveReplicationButton.png[,140,align=Left]
++
+The replication is now started.
+
+. Examine the replication.
+First, examine the *XDCR Replications* screen.
++
+image::manage-xdcr/outgoingReplicationImplicit.png[,680,align=Left]
++
+This confirms that replication is underway.
+Next, access cluster `10.144.210.103`, access its *Buckets* screen, and access the *Scopes and Collections* screen for the bucket *tsBackup*.
+Then, left-click on the *Documents* tab for the collection *airlineCollection*.
+The *Documents* screen for the collection is now displayed, and appears as follows:
++
+image::manage-xdcr/documentsImplicitlyReplicated.png[,680,align=Left]
+
+This confirms that all documents within _airlineScope_._airlineCollection_ on `10.144.210.102` have been implicitly replicated to _airlineScope_._airlineCollection_ on `10.144.210.103`.
+
+[#replicate-data-between-collections-explicitly]
+== Replicate Data Between Collections Explicitly
+
+An _explicit_ mapping between collections is one established by an administrator, so as to allow replication to occur between differently named collections, and to apply filtering.
```

Comment:
> and to apply filtering.

This makes it sound like filtering is a sub-feature of explicit mapping. 
Filtering, whenever turned on, should work in either implicit or explicit mapping (or even, without collections).

Question: What will happen to the Advanced Filtering section that you had for 6.5 in 7.0? Essentially, this filtering section is exactly the same Adv Filtering sectiont hat you worked on for 6.5, but is now located here.

By *not* having Filtering in its own subsection and having it exist under explicit mapping, my fear is that readers will have the following misconceptions:

1. Filtering can only apply to explicit mapping
2. Explicit mapping's purpose is meant for filtering documents: it is not, as its core purpose is to enable advanced mapping management.

Reply:
Okay, I will reword asap to avoid the misconception, then will put the filtering in its own subsection.